R/chat.R

```diff
@@ -132,7 +132,7 @@ Chat <- R6::R6Class(
       # Combine counts for input tokens (cached and uncached)
       tokens_acc[, 1] <- tokens_acc[, 1] + tokens_acc[, 3]
       # Then drop cached tokens counts
-      tokens_acc <- tokens_acc[, 1:2]
+      tokens_acc <- tokens_acc[,, drop = FALSE]
```

Comment:
It seems like your new code will not drop the cached tokens count which will mean that they get double counted?